library/src/scala/tasty/Reflection.scala

```diff
@@ -419,40 +419,40 @@ class Reflection(private[scala] val internal: CompilerInterface) { self =>
   //   QUOTES   //
   ////////////////
 
-  implicit class QuotedExprAPI[T](expr: scala.quoted.Expr[T]) {
+  given  QuotedExprOps: extension (expr: scala.quoted.Expr[?]) {
```

Comment:
Can we use the new syntax in #7917? Same for the changes blow.

Reply:
I'll update that syntax in a following PR

Reply:
#8156